adminSiteServer/app.tsx

```diff
@@ -101,7 +101,7 @@ export class OwidAdminApp {
         app.use("/api", publicApiRouter.router)
         app.use("/admin/api", apiRouter.router)
         app.use("/admin/test", testPageRouter)
-        app.use("/admin/assets", express.static("itsJustJavascript/webpack"))
+        app.use("/admin/assets", express.static("dist/assets"))
```

Comment:
One slight change to how this worked before is actually that this is not really used any more - instead, we always load `admin.mjs` not from the admin server but from the actual live server (e.g. tufte-owid.netlify.app).
Shouldn't make much of a difference, though.

The only slight concern is that, since `admin.mjs` is publicly accessible (which it also is today), people could find out the values of admin-only client settings like `IMAGE_HOSTING_BUCKET_PATH`.